### DISCUSSION TITLE
Split rolls across multiple machines

### DISCUSSION THREAD

**Discussion by BigPeteL** [Chia-Network/chia-blockchain#4783] [Support]
Hello Folks,

I've read the below article but it doesnt completely make it clear for my scenario - 

https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

My topology is as follows (all Ubuntu) - 
*****************/---- Farming Node 01 (GUI) ---- Plotting Node 01 (CLI)
Full Node (GUI)
*****************\ ---- Farming Node 02 (GUI) ---- Plotting Node 02 (CLI)

On the plotting nodes I have followed the above article (uPNP disabled etc), however for the farming nodes (i'd like to use the GUI) wouldnt I need to disable uPNP and point them to the Full Node for Sync/Wallet etc?  Is there any guide for something similar to the above?

Thanks!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
